extensions/shared/external-media/sources/with-media.js

```diff
@@ -37,8 +37,10 @@ export default function withMedia() {
 				};
 			}
 
-			modalRef = el => {
+			contentRef = el => {
 				if ( el ) {
+					// Store modal content.
+					this.contentElement = el;
```

Comment:
Depending on what happens about [the stopPropogation issue](https://github.com/Automattic/jetpack/pull/16118/files#r439038767), we may not need this `contentElement` at all.